udaykalra/portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/** Creates an <li> element containing text. */
+function createCommentList(inputText) {
+  const listElement = document.createElement('li');
+  listElement.innerText = inputText;
+  return listElement;
+}
 /**
- * Adds a random greeting to the page.
+ * Fetches comments for display.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function getCommentData() {
+  fetch('/data')                          // sends a request to /my-data-url
+      .then(response => response.json())  // parses the response as JSON
+      .then((myObject) => {  // now we can reference the fields in myObject!
```

Comment:
Re: the naming of the `myObject` param... can the name be more specific? What type of object is this, or what does it represent?

Reply:
Done